lib/kimono_labs_client/web_client.rb

```diff
@@ -0,0 +1,31 @@
+module KimonoLabsClient
+  # Web client for fetching a json endpoint's content via http.
+  class WebClient
+    attr_reader :api_key
+
+    # initializer
+    def initialize(api_key)
+      fail 'API key must be provided!' if api_key.nil?
+
+      @api_key = api_key
+
+      @http = Faraday.new(url: 'https://www.kimonolabs.com') do |http|
```

Comment:
Break out the instantiation of this object into a private method. That makes it easier to override for testing.